scripts/gobuild.sh

```diff
@@ -48,4 +48,7 @@ else
 		   -ldflags "-s ${VERSION_FLAG} ${GIT_HASH_FLAG} ${GIT_DIRTY_FLAG}" \
 		   -o "${TOPWD}/amazon-ecs-init"
 fi
+CGO_ENABLED=1 CGO_LDFLAGS_ALLOW='-Wl,--unresolved-symbols=ignore-in-object-files' go build -x \
```

Comment:
this does not need `CGO_ENABLED=1` or `CGO_LDFLAGS_ALLOW='-Wl,--unresolved-symbols=ignore-in-object-files'`

Reply:
this may still need `CGO_ENABLED=0`

Reply:
Fixed